radixdlt/src/main/java/com/radixdlt/CerberusModule.java

```diff
@@ -50,4 +65,23 @@ protected void configure() {
 	private PacemakerImpl pacemaker() {
 		return new PacemakerImpl(Executors.newSingleThreadScheduledExecutor());
 	}
+
+	@Provides
+	@Singleton
+	private VertexStore getVertexStore(
+		Universe universe,
+		RadixEngine radixEngine,
+		@Named("self") EUID self
+	) throws RadixEngineException {
+		if (universe.getGenesis().size() != 1) {
+			throw new IllegalStateException("Can only support one genesis atom.");
+		}
+
+		final Vertex genesisVertex = Vertex.createGenesis(universe.getGenesis().get(0));
+		final VertexMetadata genesisMetadata = new VertexMetadata(Round.of(0), genesisVertex.getId(), Round.of(0), genesisVertex.getId());

```

Comment:
Would be better if `Round.of(0)` was replaced with something like `Round.GENESIS`, or `Round.genesis()`.